test/bin/cloud_sql_proxy

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Not clear on the reasoning behind this file.

Reply:
It's a good question. Mostly i struggled to set up the actual proxy on GitHub actions for over an hour and finally just gave up and wrote a fake one in five minutes. I can add a comment though!